third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -388,6 +395,7 @@ struct FpToFpOpConversion
                     std::optional<RoundingMode> roundingMode) const {
     auto F8E4M3TyID = TypeID::get<Float8E4M3FNUZType>();
     auto F8E5M2TyID = TypeID::get<Float8E5M2Type>();
+    auto F8E5M2FNUZTyID = TypeID::get<Float8E5M2FNUZType>();
```

Comment:
I thought this type wasn't supported on NV? How is it different than F8E5M2TyID? Looks like you use the same conversion